src/main/java/org/spongepowered/api/item/EnchantmentTypes.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.item;
+
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+
+/**
+ * An enumeration of known {@link EnchantmentType}s.
+ */
+public final class EnchantmentTypes {
+
+    // SORTFIELDS:ON
+
+    /**
+     * Increases underwater mining rate.
+     *
+     * <p>In vanilla the maximum level is 1.</p>
+     */
+    public static final EnchantmentType AQUA_AFFINITY = DummyObjectProvider.createFor(EnchantmentType.class, "AQUA_AFFINITY");
+
+    /**
+     * Increases damages to arthropod mobs. In vanilla this includes spiders,
+     * cave spiders, silverfish, and endermites.
+     *
+     * <p>In vanilla the maximum level is 5.</p>
```

Comment:
At high level, the enchantment will slow down the arthropod hit for few seconds.

Reply:
Interesting, never knew this.